build-log-server.sh

```diff
@@ -1,24 +1,22 @@
 #!/bin/bash
 
-SCRIPT=$(readlink -f ${BASH_SOURCE[0]})
-SCRIPTPATH=`dirname "$SCRIPT"`
-MYDIRBLOCK=$SCRIPTPATH
-CURRWORKINGDIR=`pwd`
+SCRIPTDIR="$( cd -P "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+WORKINGDIR="`pwd`"
 
```

Comment:
That would be the most convenient thing to use, but it doesn't ship with an Ubuntu base install (even though the man-page does oddly enough). 

So it's a trade-off between adding another dependency just to determine the script path, or having that jumble of system commands which are rather ugly but always available.

I defaulted to the latter, but I don't mind changing it... which would you prefer?